Userland/Libraries/LibIPC/Connection.h

```diff
@@ -48,13 +48,18 @@ class Connection : public Core::Object {
     }
 
     void post_message(const Message& message)
+    {
+        post_message(message.encode());
+    }
+
+    // FIXME: unnecessary copy
+    void post_message(MessageBuffer buffer)
```

Comment:
Why not take a reference then?
what's being copied here exactly anyway (and why is this public to begin with?)